src/native/external/llvm-libunwind/src/AddressSpace.hpp

```diff
@@ -117,23 +83,12 @@ namespace libunwind {
 //   __eh_frame_hdr_start = SIZEOF(.eh_frame_hdr) > 0 ? ADDR(.eh_frame_hdr) : 0;
 //   __eh_frame_hdr_end = SIZEOF(.eh_frame_hdr) > 0 ? . : 0;
 
-#ifndef _LIBUNWIND_USE_ONLY_DWARF_INDEX
```

Comment:
`_LIBUNWIND_USE_ONLY_DWARF_INDEX`, `_LIBUNWIND_BAREMETAL_DWARF_INDEX_SEC_START` and `_LIBUNWIND_BAREMETAL_DWARF_INDEX_SEC_END` were added in https://github.com/dotnet/corert/pull/8271, but they are always using the default value, and not defined anywhere else. I have removed them as part of resolving merge conflicts (and to avoid future conflicts).